.github/workflows/ci_test.yml

```diff
@@ -3,34 +3,12 @@ on: [push, pull_request]
 name: CI test and lint
 
 jobs:
-  check:
-    name: Check
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout sources
-        uses: actions/checkout@v2
-
-      - name: Install stable toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: stable
-          override: true
-
-      - name: Run cargo check
```

Comment:
I think the check command is still useful and should stay. It runs much quicker than for example test, and you can quickly try all combinations of features.

Reply:
Sure, I'll add it back if you want it. All though note that I believe clippy and check does roughly the same things right now (and both take ~1 minute to run).

I think `args: --no-default-features` is missing below which is what confused me, so I'll add that too.